src/proxy/CMakeLists.txt

```diff
@@ -15,16 +14,14 @@
 #  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 if(WITH_XC_BROWSER)
-    include_directories(${BROWSER_SOURCE_DIR})
-
     set(proxy_SOURCES
-        ../core/Alloc.cpp
```

Comment:
I suppose you removed this because we're linking statically?

Reply:
No I should not have removed this file, adding back in. Good catch.